<?xml version="1.0"?>
<data>
	<input name='x_axis'>
		<type>float</type>
	</input>
	<input name='y_axis'>
		<type>float</type>
	</input>
	<input name='z_axis'>
		<type>float</type>
	</input>
</data>